tests/main/interfaces-desktop-launch/test-app/bin/app.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/sh
+
+echo "App launched"
+
+cat << EOF > $SNAP_USER_DATA/launch-data.txt
```

Comment:
Looks like I tripped up the static checks in the spread test.  To pass shell check, change this to:
```suggestion
cat << EOF > "$SNAP_USER_DATA"/launch-data.txt
```